handlers/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: Run postmap
+  command: postmap /etc/postfix/{{ file['name'] }}
+  when: file['postmap'] | default(false) | bool
+  no_log: "{{ postfix_no_log }}"
+  loop: "{{ postfix_files }}"
+  loop_control:
+    loop_var: file
+  changed_when: false
```

Comment:
This task results in a change, is it? Is there a way to make it idempotent?

Reply:
Sadly, no, there's no output for postmap and the exit code is always 0 (when it succeeds)

```
root@debian:~# touch file
root@debian:~# postmap file # creating a new file.db file
root@debian:~# echo $?
0
root@debian:~# postmap file # updates the existing file.db file
root@debian:~# echo $?
0
```

Reply:
Can it list created files?

Reply:
I think we should not do this in a handler - see below